web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v370/migrate-default.sql

```diff
@@ -4,6 +4,8 @@ DELETE FROM Settings WHERE name = 'ui/config';
 
 ALTER TABLE Sources DROP islocal;
 
+UPDATE Metadata SET data = replace(data, 'http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/codelist/ML_gmxCodelists.xml', 'http://standards.iso.org/iso/19139/resources/gmxCodelists.xml') WHERE data LIKE '%http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/codelist/ML_gmxCodelists.xml%' AND schema = 'iso19139';
```

Comment:
Should not be updated any schema based on `iso19139`?

Reply:
Probably if it target INSPIRE compliance, but we can't really know it from here. This is to the responsibility of the profile no ?